modules/op-configuring-pipelines-as-code-logging.adoc

```diff
@@ -0,0 +1,151 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-pipelines-as-code-logging_{context}"]
+= Configuring {pac} logging 
+
+You can configure {pac} logging by editing the `pac-config-logging` config map in the `TektonConfig` custom resource (CR).
+
+.Prerequisites
+
+* {pac} is installed on your cluster.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Administration* → *CustomResourceDefinitions*.
+
+. Use the *Search by name* box to search for the `tektonconfigs.operator.tekton.dev` custom resource definition (CRD) and click *TektonConfig* to see the CRD details page.
+
+. Click the *Instances* tab.
+
+. Click the *config* instance to see the `TektonConfig` CR details.
+
+. Click the *YAML* tab.
+
+. Edit the `loglevel.` fields under the `.options.configMaps.pac-config-logging.data` parameter based on your requirements.
++
+.Example: `TektonConfig` CR with {pac} log level fields set to `warn`
```

Comment:
I have never seen this phrased this way. I am wondering if this might work better? 
```suggestion
.Sample YAML of `TektonConfig` CR with {pac} log level fields set to `warn`
```

Reply:
@opayne1 We actually had this discussion before in the team-ocp-docs. In Pipelines documentation we use the "Example: Description" format almost every time so I wanted it to stay consistent with the rest of the documentation. But the "Example of <description.>" is also occasionally used.